pkg/kyverno/common/common.go

```diff
@@ -0,0 +1,149 @@
+package common
+
+import (
+	"bufio"
+	"bytes"
+	"encoding/json"
+	"errors"
+	"fmt"
+	"io"
+	"io/ioutil"
+	"os"
+	"path/filepath"
+
+	v1 "github.com/nirmata/kyverno/pkg/api/kyverno/v1"
+	"github.com/nirmata/kyverno/pkg/kyverno/sanitizedError"
+	"github.com/nirmata/kyverno/pkg/openapi"
+	"k8s.io/apimachinery/pkg/util/yaml"
+)
+
+//GetPolicies - Extracting the policies from multiple YAML
+func GetPolicies(paths []string) ([]*v1.ClusterPolicy, error) {
+	var policies []*v1.ClusterPolicy
+	for _, path := range paths {
+		path = filepath.Clean(path)
+
+		fileDesc, err := os.Stat(path)
+		if err != nil {
+			return nil, err
+		}
+
+		if fileDesc.IsDir() {
+			files, err := ioutil.ReadDir(path)
+			if err != nil {
+				return nil, err
+			}
+
+			listOfFiles := make([]string, 0)
+			for _, file := range files {
+				listOfFiles = append(listOfFiles, filepath.Join(path, file.Name()))
+			}
+
+			policiesFromDir, err := GetPolicies(listOfFiles)
+			if err != nil {
+				return nil, err
```

Comment:
same. please check other returns as well to see if it would help to log errors.

Since we are building a CLI, the error logging can be at a higher level and we should return the error back to the user. The CLI should exit with a non-zero value if there is an error, to allow it to be used from other scripts, etc. which check returns.

Reply:
Done.